models/event-subEvent-patch.json

```diff
@@ -98,4 +97,4 @@
       "id"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Nit: The Stoplight Editor will remove this again next time so this will be a never-ending battle sadly

Reply:
Ok, good to know. I added it because the 2 other files already had a newline and this one not.